app-editors/dav/Manifest

```diff
@@ -1 +1,2 @@
 DIST dav-0.8.5.tar.gz 22824 BLAKE2B 969458aa6b914f9cd624554690a88f4c2999dcc6e89db38d13d9ba98db7881a34ce1575ccbdf74b3ecca34345f2ec639cfc805a350e31f856f7d086cf23752de SHA512 cdedff875df392e8525399b40d45cdf4e984b1b014f03d9f5cd5be270eebe0003fcf48e9738f6ac2789138973970bb9fe6d0b8babd20c1df61e193f84ee066b3
+DIST dav-0.9.0.tar.gz 23882 BLAKE2B 6845b40bbb74d57438625c2736758c52fbe4cff3e020fd451dee540f06fb893d71e11469f828845c193e9fbbd8a8e0b31dcb9a6fa29383053b61dd9a9d7b0ce4 SHA512 8bb9d111c25e7e41c7d75ee559aeae47494ff68f19f40d2ba3e2ebca23577538b94ba72b58ccf4e64ef92d9e740ee843fa249ef0e4148971b3037259b4f280c5
```

Comment:
You have to regenerate manifest, it does not fit the archive name after last push.